### PR TITLE
upgrade to boost 1.69.0

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2684/

**Comment by jsteemann** [comment]
Previous build compiled only on Linux, but not Windows and Mac.
Retrying after some fixes:
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2686/

File: 3rdParty/CMakeLists.txt [hunk 1]
```diff
@@ -114,11 +114,11 @@ set(ARANGO_S2GEOMETRY_VERSION "${ARANGO_S2GEOMETRY_VERSION}" PARENT_SCOPE)
 if (USE_IRESEARCH)
   add_library(boost-static IMPORTED STATIC)
   set_target_properties(boost-static PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/boost/1.62.0"
+    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/boost/1.69.0"
```
Comment (dothebart) [inline 3rdParty/CMakeLists.txt]: DRY please. pull the version up. most probably once more in boost/CMakeFile too?

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2694/

**Comment by jsteemann** [comment]
@dothebart: build is blue, do you want to review my last changes?

**Pull request merged**